.gitignore

```diff
@@ -67,3 +67,7 @@ cmake-build*
 # CBMC files
 *.goto
 *.log
+
+# Rust and Cargo
+*.lock
+aws_crt_common_ws/target
```

Comment:
probably want `target/` — if you build in a submodule you may get another target dir

Reply:
+1